mainapp/models.py

```diff
@@ -165,6 +165,7 @@ class Contributor(models.Model):
     phone = models.CharField(max_length=10)
     address = models.TextField()
     commodities = models.TextField(verbose_name="What you can contribute. ( സംഭാവന ചെയ്യാന്‍ ഉദ്ദേശിക്കുന്ന സാധനങ്ങള്‍ ) -- Eg: Shirts, torches etc ")
+    blood_group=models.TextField()
```

Comment:
A `CharField` with `max_length=5` will be enough for blood group. TextFields are for large strings

Reply:
well , that's done now. Thanks !